packages/react/ds/src/textarea/textarea.tsx

```diff
@@ -1,38 +1,65 @@
 import React from 'react';
+import { ErrorText, ErrorTextProps } from '../error-text/error-text.js';
+import { HintText, HintTextProps } from '../hint-text/hint-text.js';
+import { Label, LabelProps } from '../label/label.js';
 
 // Extend `React.TextareaHTMLAttributes<HTMLTextAreaElement>` so that
 // the component can accept all the standard attributes and events that a `<textarea>` element can handle.
 export type TextAreaProps =
   React.TextareaHTMLAttributes<HTMLTextAreaElement> & {
     ref?: React.Ref<HTMLTextAreaElement>;
-    hasError?: boolean;
     rows?: number;
     cols?: number;
     autoComplete?: string;
+    errorText?: ErrorTextProps;
+    hintText?: HintTextProps;
+    label?: LabelProps;
   };
 
 // Use React.forwardRef to support refs properly
 export const TextArea = React.forwardRef<HTMLTextAreaElement, TextAreaProps>(
   (
     {
-      hasError,
       rows = 4, // default row count
       cols = 100, // default column count
       autoComplete = 'on', // default autoComplete behavior
+      label,
+      errorText,
+      hintText,
+      id,
       ...props
     },
     ref,
   ) => {
+    const hasError = errorText && errorText.content;
```

Comment:
I think there's something I'm missing here. You're checking to see if the `content` property exists but in the `ErrorText` component we don't use a `content` prop. I checked to see if it works and it does but not sure where it is picking up the value from

Reply:
Good catch! Fixed now.